lib/hashid/rails.rb

```diff
@@ -84,10 +84,13 @@ def find_by_hashid!(hashid)
         find_by!(id: decode_id(hashid, fallback: false))
       end
 
+      def hashid_table_name
```

Comment:
Is there a compelling reason for making this public? I'd prefer it kept private since it's for internal use by the gem.

Reply:
Not necessarily `public`, although, I think `protected` might be a better choice as the idea is that you override the method when/if you need to change the original class name.